source/migrate-hosting-graphql.txt

```diff
@@ -56,6 +57,21 @@ products and features.
 
 Refer to :ref:`Migrate GraphQL to Hasura <migrate-hasura>` for details.
 
+WunderGraph
+~~~~~~~~~~~
+
+WunderGraph is a comprehensive developer platform that comes equipped with an
+extensive variety of open-source tools and Software Development Kits (SDKs).
```

Comment:
```suggestion
extensive variety of open-source tools and software development kits (SDKs).
```
or just `SDKs` since that seems acceptable to abbreviate every time in our docs